test/tests/fvbcs/fv_neumannbc/tests

```diff
@@ -0,0 +1,28 @@
+[Tests]
+  [fv_neumann]
+    design = 'FVNeumannBC.md'
+    issues = '#16477'
+    type = 'Exodiff'
+    input = 'fv_neumannbc.i'
+    exodiff = 'fv_neumannbc_out.e'
+    requirement = 'The system shall run a simple 1D diffusion problem with a Neumann BC value.'
+  []
+  [fvbcs_internal]
+    design = 'FVNeumannBC.md'
+    issues = '#16882'
+    type = RunException
+    input = 'fv_neumannbc.i'
+    cli_args = "FVBCs/inactive=''"
+    expect_err = 'A FVFluxBC is being triggered on an internal face with centroid'
```

Comment:
```suggestion
    expect_err = 'A FVFluxBC is being triggered on an internal face'
```

Reply:
I don't know if I love the current message with the centroid reporting, but I don't have anything better right now. However, in the future I could see us modifying the error a bit and maybe not printing the centroid (e.g. we might just print the adjoining subdomain ids or something)

Reply:
I would be fine with adjoining subdomain id, I just would hate to have the surface id, which is harder to track down.